util.go

```diff
@@ -38,7 +38,7 @@ func encodeQuery(query map[string][]string) string {
 			continue
 		}
 
-		tuple := key + "=" + oDataEncodeVals(vals)
+		tuple := key + "=" + strings.Replace(strings.Join(vals, ","), " ", "%20", -1)
```

Comment:
Why was this over-encoding? don't we *want* an OData encoding? (@Page- will probably know better if this will work)